recipes/pcl/all/conanfile.py

```diff
@@ -0,0 +1,577 @@
+        tc.cache_variables["WITH_PCAP"] = self._is_enabled("pcap")
+        tc.cache_variables["WITH_PNG"] = self._is_enabled("png")
+        tc.cache_variables["WITH_QHULL"] = self._is_enabled("qhull")
+        tc.cache_variables["WITH_QT"] = self._is_enabled("qt")
+        tc.cache_variables["WITH_VTK"] = self._is_enabled("vtk")
+        tc.cache_variables["WITH_CUDA"] = self._is_enabled("cuda")
+        tc.cache_variables["BUILD_CUDA"] = self._is_enabled("cuda")
+        tc.cache_variables["BUILD_GPU"] = self._is_enabled("cuda")
+        tc.cache_variables["WITH_SYSTEM_ZLIB"] = True
+        tc.cache_variables["PCL_ONLY_CORE_POINT_TYPES"] = self.options.precompile_only_core_point_types
+        # The default False setting breaks OpenGL detection in CMake
+        tc.cache_variables["PCL_ALLOW_BOTH_SHARED_AND_STATIC_DEPENDENCIES"] = True
+        tc.variables["OpenGL_GL_PREFERENCE"] = "GLVND"
+
+        if not self.options.add_build_type_postfix:
+            tc.cache_variables["CMAKE_DEBUG_POSTFIX"] = ""
+            tc.cache_variables["CMAKE_RELEASE_POSTFIX"] = ""
+            tc.cache_variables["CMAKE_RELWITHDEBINFO_POSTFIX"] = ""
+            tc.cache_variables["CMAKE_MINSIZEREL_POSTFIX"] = ""
+
+        tc.cache_variables["BUILD_tools"] = self.options.tools
+        tc.cache_variables["BUILD_apps"] = self.options.apps
+        tc.cache_variables["BUILD_examples"] = False
+        enabled = sorted(self._enabled_components())
+        disabled = sorted(self._disabled_components())
+        self.output.info("Enabled components: " + ", ".join(enabled))
+        self.output.info("Disabled components: " + ", ".join(disabled))
+        for comp in enabled:
+            tc.cache_variables[f"BUILD_{comp}"] = True
+        for comp in disabled:
+            tc.cache_variables[f"BUILD_{comp}"] = False
+
+        tc.generate()
+
+        deps = CMakeDeps(self)
+        deps.set_property("eigen", "cmake_file_name", "EIGEN")
+        deps.set_property("flann", "cmake_file_name", "FLANN")
+        deps.set_property("flann", "cmake_target_name", "FLANN::FLANN")
+        deps.set_property("libpcap", "cmake_file_name", "PCAP")
+        deps.set_property("qhull", "cmake_file_name", "QHULL")
+        deps.set_property("qhull", "cmake_target_name", "QHULL::QHULL")
+        deps.generate()
+
+        deps = PkgConfigDeps(self)
+        deps.generate()
+
+    def _patch_sources(self):
+        apply_conandata_patches(self)
+        for mod in ["Eigen", "FLANN", "GLEW", "Pcap", "Qhull", "libusb"]:
+            os.remove(os.path.join(self.source_folder, "cmake", "Modules", f"Find{mod}.cmake"))
+
+    def build(self):
+        self._patch_sources()
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def package(self):
+        copy(self, "LICENSE.txt",
+             dst=os.path.join(self.package_folder, "licenses"),
+             src=self.source_folder)
+
+        cmake = CMake(self)
+        cmake.install()
```

Comment:
Looking at https://c3i.jfrog.io/c3i/misc/logs/pr/18389/29-windows-visual_studio/pcl/1.13.1//f36e8de945c19577931147f7f0c36bdeef6741dc-build.txt we see that [PDB FILES NOT ALLOWED (KB-H017)] check is OK and we don't see pdb files installed by cmake install.
But I can add the missing line from the template to prevent PDBs installation.

Thanks for your review.